examples/connext_dds/crypto_algorithms/c/CMakeLists.txt

```diff
@@ -0,0 +1,25 @@
+# (c) 2022 Copyright, Real-Time Innovations, Inc. All rights reserved.
+# No duplications, whole or partial, manual or electronic, may be made
+# without express written permission.  Any such copies, or revisions thereof,
+# must display this notice unaltered.
+# This code contains trade secrets of Real-Time Innovations, Inc.
+cmake_minimum_required(VERSION 3.11)
+project(rtiexamples-crypto-algorithms)
+
+set(CMAKE_MODULE_PATH
+    ${CMAKE_MODULE_PATH}
+    "${CMAKE_CURRENT_SOURCE_DIR}/../../../../resources/cmake"
+)
+
+# Include ConnextDdsAddExample.cmake from resources/cmake
+include(ConnextDdsAddExample)
+
+connextdds_add_example(
```

Comment:
[Luis] The connextdds_add_example is only going to call the findpackage with the core libraries. The security plugins library should also be included.